packages/esm-form-entry-app/src/app/form-creation/form-creation.service.ts

```diff
@@ -111,6 +113,10 @@ export class FormCreationService {
     this.dataSources.registerDataSource('patient', this.formDataSourceService.getPatientObject(patient), true);
     this.dataSources.registerDataSource('rawPrevEnc', createFormParams.previousEncounter, false);
     this.dataSources.registerDataSource('userLocation', createFormParams.user.sessionLocation);
+    this.dataSources.registerDataSource(
```

Comment:
So, the data source, in this case, should be an object containing this function, not the result of the function call